### PR TITLE
chore: codecov update target for patch/project

### PR DESCRIPTION
**Pull request by RemiBonnet**
# What does this PR do?

- Update Codecov target 

---

## PR Checklist

- [ ] This PR introduces breaking change(s) and has been labeled as such
- [ ] This PR introduces new store changes
- [x] I have followed the library pattern i.e `feature`, `ui`, `data`, `utils`
- [x] I made sure the code is type safe (no any)

### CODE REVIEW

**Comment by acarranoqovery** [comment]
[Qovery Preview](https://www.qovery.com)
---

A preview environment was automatically created via Qovery.
Click on the link below to follow its deployment and use it.
:point_right: [\[PR\] staging - chore: codecov update target for patch/project - 2023-08-04T13:07:17Z](https://console.qovery.com/organization/3d542888-3d2c-474a-b1ad-712556db66da/project/a64f6cc1-4e34-433b-b8fd-5359716fb01b/environment/ac7097d7-e652-414d-8639-a0d27ee9ba8b/services/general)

 Another comment will be posted when deployments are terminated

**Comment by acarranoqovery** [comment]
[Qovery Preview](https://www.qovery.com)
---

Your preview environment failed to be deployed !
Click on the link below to check the deployment logs.
:point_right: [\[PR\] staging - chore: codecov update target for patch/project - 2023-08-04T13:07:17Z](https://console.qovery.com/organization/3d542888-3d2c-474a-b1ad-712556db66da/project/a64f6cc1-4e34-433b-b8fd-5359716fb01b/environment/ac7097d7-e652-414d-8639-a0d27ee9ba8b/logs)

**Review by ctjhoa — APPROVED** [review]

File: .codecov.yml [hunk 1]
```diff
@@ -2,4 +2,7 @@ coverage:
   status:
     project:
       default:
-        threshold: 1%
+        target: 55%
+    patch:
+      default:
+        target: 60%
```
Comment (ctjhoa) [inline .codecov.yml:8]:
```suggestion
        target: auto
    patch: off
```

**Review by ctjhoa — CHANGES_REQUESTED** [review]
Reading more about codecov
https://about.codecov.io/blog/the-5-levels-of-code-coverage-how-to-build-a-testing-culture-in-your-organization/

I think this is more relevant for our usage for now.

**Review by ctjhoa — APPROVED** [review]

**Pull request merged**